book/src/monitor/api.md

```diff
@@ -0,0 +1,177 @@
+# Monitor API
+
+<style>
+  var { color: var(--sidebar-fg); background: var(--sidebar-bg); padding: 0 0.25em; margin: 0 0.125em; }
+  dt { margin-top: 0.5em; font-weight: bold; }
+  h3 ._method { margin-right: 1em; }
+</style>
+
+- [Reserving runners](#reserving-runners)
+  - [<span class="_method">POST</span> /select-runner](#POST/select-runner)
+  - [<span class="_method">POST</span> /profile/<var>profile_key</var>/take](#POST/profile/.../take)
+  - [<span class="_method">POST</span> /profile/<var>profile_key</var>/take/<var>count</var>](#POST/profile/.../take/...)
+- [Runner internals](#runner-internals)
+  - [<span class="_method">GET</span> /github-jitconfig](#GET/github-jitconfig)
+  - [<span class="_method">GET</span> /boot](#GET/boot)
+- [Dashboard internals](#dashboard-internals)
+  - [<span class="_method">GET</span> /dashboard.html](#GET/dashboard.html)
+  - [<span class="_method">GET</span> /dashboard.json](#GET/dashboard.json)
+  - [<span class="_method">GET</span> /profile/<var>profile_key</var>/screenshot.png](#GET/profile/.../screenshot.png)
+  - [<span class="_method">GET</span> /runner/<var>runner_id</var>/screenshot.png](#GET/runner/.../screenshot.png)
+  - [<span class="_method">GET</span> /runner/<var>runner_id</var>/screenshot/now](#GET/runner/.../screenshot/now)
+- [Policy overrides (EXPERIMENTAL)](#policy-overrides-experimental)
+  - [<span class="_method">GET</span> /policy/override](#GET/policy/override)
+  - [<span class="_method">POST</span> /policy/override](#POST/policy/override)
+  - [<span class="_method">DELETE</span> /policy/override](#DELETE/policy/override)
+
+## Notes about endpoints
+
+Some of the endpoints below **require the monitor API token**.
+Requests to these endpoints need to prove you know that token:
+
+```
+Authorization: Bearer <monitor API token>
+```
+
+Some of the endpoints below **may require cranking the backend**.
```

Comment:
This phrase was not immediately clear to me (I am not native speaker), but maybe we can reword it into something simpler and more explanatory:
> may involve sequential processing in the backend

or 
> may incur heavier backend load